### PR TITLE
Add "camelize" flag to the model generator

### PR DESCRIPTION
**Pull request by izabolotnev**
Add "camelize" flag to the model generator. Simple add --camelize and model`s properies will be named  in lowerCamelCase style.

### CODE REVIEW

File: scripts/Phalcon/Builder/Model.php [hunk 1]
```diff
@@ -255,63 +259,13 @@ public function build()
...
-                        'belongsTo',
-                        $relation['fields'],
-                        $entityNamespace . $entityName,
-                        $relation['relationFields'],
-                        $this->snippet->getRelationOptions(isset($relation['options']) ? $relation["options"]->toArray() : null)
-                    );
-                }
-            }
-        }
-
```
Comment (sergeyklay) [inline scripts/Phalcon/Builder/Model.php]: why you removed this?

Comment (izabolotnev) [inline scripts/Phalcon/Builder/Model.php]: Model generator hasn`t options hasMany and belongsTo. This code was unreachable in current realisation.

Comment (sergeyklay) [inline scripts/Phalcon/Builder/Model.php]: Ah, yes, I see

**Comment by sergeyklay** [comment]
Thanks

**Pull request merged**